Shared/Samples/Monitor changes to draw status/README.md

```diff
@@ -0,0 +1,28 @@
+# Monitor changes to draw status
+
+Get the draw status of your map view or scene view to know when all layers in the map or scene have finished drawing.
+
+![Image of Monitor changes to draw status](monitor-changes-to-draw-status.png)
+
+## Use case
+
+You may want to display a loading indicator while layers are loading, which could then be removed on `completed`.
```

Comment:
> which could then be removed on `completed`.

This seems to be an incomplete thought. What is completed? Instead, you could say something like "when all the layers have finished drawing" or "when the draw status for each layer is `completed`".